src/utils/frameOfReference.ts

```diff
@@ -88,3 +90,60 @@ export function getSmallestSpacing(
   spacing.splice(axisIndex, 1);
   return Math.min(...spacing);
 }
+
+export function toFromPlane({ planeOrigin, planeNormal }: FrameOfReference) {
```

Comment:
This name sounds confusing, since it can read as "to 'from plane'" as well as "to/from plane". Maybe `getPlaneTransforms` or something similar?

Reply:
renamed to getPlaneTransforms